README.md

```diff
@@ -137,11 +137,10 @@ var escapeRegExp = s =>
 
 ### Even or odd number
 
-Use `Math.abs()` to extend logic to negative numbers, check using the modulo (`%`) operator.
```

Comment:
30 seconds of code is primarily a learning resource. Kindly explain how this works in one line so that people understand. I like the idea though. Also, you should apply changes to the individual snippet file, not README.md, as the latter is auto-generated by a script and changes in it will be overwritten (yes, contribution guidelines were not clear, I will update them).

Reply:
Working on it